### DISCUSSION TITLE
Cloud instances: clarification about embedded impacts in verbose mode

### DISCUSSION THREAD

**Discussion by demeringo** [Boavizta/boaviztapi#221] [Q&A] [answered]
Hi,

When retrieving impacts (using cloud-scanner) we would like to retrieve embedded impacts of each component separately.

1. When using the non-verbose mode, we retrieve aggregated impacts of the instance (i.e. the impacts of the underlying server divided by the number of instances per server).

2. The **verbose** output of the cloud routes allow to get more detail about the components but my understanding is that impacts are returned for the _entire_ server hosting the VM and that no ratio (number of instance per server) is applied.

Can you confirm that 2 is true ?

We would like to attribute only a part of embedded impacts of each component to the instance; https://github.com/Boavizta/cloud-scanner/issues/333

Would it be correct to divide theses verbose impacts by the `usage\instance_per_server` value that we can retrieve in the cloud archetype route ?

**Comment by da-ekchajzer** [comment, marked as answer]
Yes, it would be correct.

**Reply by demeringo** [reply]
Thanks @da-ekchajzer

**Comment by da-ekchajzer** [comment]
Implemented in the latest version of the API (v1.2.1)
